rtl/connector.py

```diff
@@ -9,22 +9,25 @@
 
 Written by Marko Kosunen 20190109 marko.kosunen@aalto.fi
 """
-import os
-from thesdk import *
+import time
+import regex as re
```

Comment:
Why is there a need to use external regex library instread of the built-in `re`?